tests/e2e/events_test/events_test.go

```diff
@@ -0,0 +1,72 @@
+// Staff tries to make calls to events endpoints
+package tests
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/HackIllinois/api/common/configloader"
+	event_models "github.com/HackIllinois/api/services/event/models"
+	"github.com/HackIllinois/api/tests/common"
+	"github.com/dghubble/sling"
+	"gopkg.in/mgo.v2"
+)
+
+var admin_client *sling.Sling
+var session *mgo.Session
+
+func TestMain(m *testing.M) {
+
+	cfg, err := configloader.Load(os.Getenv("HI_CONFIG"))
+
+	if err != nil {
+		fmt.Printf("ERROR: %v\n", err)
+		os.Exit(1)
+	}
+
+	admin_client = common.GetSlingClient("Admin")
```

Comment:
Is the objective to test Admin role privileges, or Staff role privileges? For now, the privileges overlap completely, so it shouldn't be a big deal right now.

Reply:
My bad. I didn't realize Admin and Staff were different roles. I'll make this change and also the flow that you suggested above.